app/src/main/java/io/appium/uiautomator2/model/internal/Gestures.java

```diff
@@ -33,43 +32,70 @@
 import static io.appium.uiautomator2.utils.ReflectionUtils.getMethod;
 import static io.appium.uiautomator2.utils.ReflectionUtils.invoke;
 
+import io.appium.uiautomator2.utils.ReflectionUtils;
+
 public class Gestures {
-    private final Object wrappedInstance;
+    private final Class<?> wrappedClass;
+    private final int displayId;
 
-    Gestures(Object wrappedInstance) {
-        this.wrappedInstance = wrappedInstance;
+    Gestures(int displayId) {
+        this.displayId = displayId;
+        // https://androidx.tech/artifacts/test.uiautomator/uiautomator/2.3.0-beta01-source/androidx/test/uiautomator/Gestures.java.html
```

Comment:
I have compared the diff, 2.2.0 and 2.3.0, so they started considering displayId as a big diff. Interesting